paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -428,6 +438,8 @@ internal abstract class BaseSheetViewModel(
         _notesText.value = text
     }
 
+    abstract fun handleSelected(selection: PaymentSelection?)
```

Comment:
The decision about what to do in case of a selection now lies in the view model, not in the fragments.